cnf-app-mac-operator/internal/controller/cnfappmac_controller.go

```diff
@@ -430,7 +441,7 @@ func getContainerLogValue(podName, namespace string) (string, error) {
 	cmd := []string{
 		"sh",
 		"-c",
-		"egrep 'Port [0-9]:' /var/log/testpmd/app.log | sed 's/Port [0-9]: //'",
+		"egrep '^Port [0-9]: ([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})$' /var/log/testpmd/app.log | sed 's/Port [0-9]: //'",
```

Comment:
Nice catch, I was reading that this regex could also match octets separated either by colon or dash, for example: 
```
manuel@fedora:~$ echo "3D-F2-C9:A6-B3:4F" | egrep '([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})$'
3D-F2-C9:A6-B3:4F
```
So maybe we can be a little bit more strict and allow only MACs with octets separated by colons only:
```
manuel@fedora:~$ echo "3D-F2-C9:A6-B3:4F" | egrep '([0-9A-Fa-f]{2}[:]){5}([0-9A-Fa-f]{2})$'
manuel@fedora:~$ echo "3D:F2:C9:A6:B3:4F" | egrep '([0-9A-Fa-f]{2}[:]){5}([0-9A-Fa-f]{2})$'
3D:F2:C9:A6:B3:4F
```
Other than that, looks good to me

Reply:
sure, let's do that